scripts/evaluate_model_entropy.py

```diff
@@ -0,0 +1,78 @@
+"""Script to evaluate learned predicate classifiers on held-out test cases."""
+
+from typing import Sequence
+
+import numpy as np
+
+from predicators.scripts.evaluate_predicate_classifiers import main
+from predicators.src import utils
+from predicators.src.approaches.interactive_learning_approach import \
+    InteractiveLearningApproach
+from predicators.src.envs import BaseEnv
+from predicators.src.settings import CFG
+from predicators.src.structs import Object, State
+
+
+def evaluate_pred_ensemble(env: BaseEnv,
+                           approach: InteractiveLearningApproach) -> None:
+    """Prints entropy and BALD scores of predicate classifier ensembles."""
+    if CFG.env == "cover":
+        return _evaluate_pred_ensemble_cover(env, approach)
+    raise NotImplementedError(
+        f"Held out predicate test set not yet implemented for {CFG.env}")
+
+
+def _evaluate_pred_ensemble_cover(
+        env: BaseEnv, approach: InteractiveLearningApproach) -> None:
+    preds = approach._get_current_predicates()  # pylint: disable=protected-access
+    Covers = [p for p in preds if p.name == "Covers"][0]
+    # Create initial state
+    task = env.get_test_tasks()[0]
+    state = task.init
+    block0 = [b for b in state if b.name == "block0"][0]
+    block1 = [b for b in state if b.name == "block1"][0]
+    target0 = [b for b in state if b.name == "target0"][0]
+    target1 = [b for b in state if b.name == "target1"][0]
+    robot = [b for b in state if b.name == "robby"][0]
+    blocks = [block0, block1]
+    targets = [target0, target1]
+    block_poses = [0.15, 0.605]
+    target_poses = [0.375, 0.815]
+    for block, pose in zip(blocks, block_poses):
+        # [is_block, is_target, width, pose, grasp]
+        state.set(block, "pose", pose)
+        # Make sure blocks are not held
+        state.set(block, "grasp", -1)
+    for target, pose in zip(targets, target_poses):
+        # [is_block, is_target, width, pose]
+        state.set(target, "pose", pose)
+    state.set(robot, "hand", 0.0)
+    # Test 1: block and target far apart
```

Comment:
If these are the same states more or less as in the other script, can we refactor to define them only once?